src/components/section/Home/TechnologiesSection.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {HeaderSection} from "../../header";
+import {Carousel} from "../../carousel";
+import {technologies} from "../../../constants/data";
+import {CardTechnologie} from "../../card";
+import {LayoutSection} from "../../layout";
+
+const TechnologiesSection = ({className}) => {
+
+    const classes = classNames(
+        "mt-32 mb-32"
+    )
+
+    return (
+        <LayoutSection>
+            <HeaderSection title={"Les technologie avec lequel je travail"} description="Pour être un bon developpeur, il faut avoir des compétence dans tout les domaine" className="w-1/3"/>
+            <Carousel className="mt-16">
+                {technologies.map( (item,index) => (
+                    <CardTechnologie
+                        key={index} title={item.title}
```

Comment:
Ne jamais utiliser l'index du map, c'est une mauvaise pratique, je te laisse relire la doc https://fr.reactjs.org/docs/lists-and-keys.html